### ISSUE TITLE
Share feature on new species discovery

### ISSUE THREAD

**Issue by olivierlacan** [inaturalist/SeekReactNative#338]
Every time I discover a new species I really want to share it with folks on Twitter in order to help them discover Seek & iNaturalist but there's no way to get a permalink to an OpenGraph friendly URL from within the app. 

I understand one goal of the app is to be kid-friendly so I'm not looking for any deep integration with any particular social media platform, only a permalink with a friendly URL that can easily be shared and doesn't contain any user-specific information, just the species details and all the cool stuff that this page already contains.

Ideally a simple nudge to sign up to iNaturalist or download Seek on the relevant app stores would be a good idea as well to encourage user sign ups, but I don't think that should be the main focus of this feature.

**Comment by albullington** [comment]
@olivierlacan I'm closing this issue because we already have a social sharing feature built into Seek. It's available for users who are logged into iNaturalist, which is the only way we can include links *and* be kid-friendly.

If you're logged in, you can tap the sharing icon to the right of the species photos, which will bring you to a screen that lets you download a watermarked photo for social or share directly to Facebook. We plan to support more social media platforms in the future.

Separately, we do also have a card on the home screen of Seek which encourages users to sign up for iNaturalist -- we're updating this to make it more prominent in this week's release.

**Comment by olivierlacan** [comment]
@albullington I think you might want to reopen this issue as a bug report because I’m signed into iNaturalist on Seek and on the latest iOS version. There is no such sharing feature visible anywhere on the observation page. 

The only link on the page is to Wikipedia. 

![98244395-F207-4E18-80EA-AAE57BCBD182](https://user-images.githubusercontent.com/65950/119282847-1e0f9f80-bc09-11eb-8b13-35a1a30b1334.png)
![E204670B-B4B2-4A51-88A6-16677EF770DD](https://user-images.githubusercontent.com/65950/119282851-2071f980-bc09-11eb-89c4-09cfa99620fb.png)
![0E6EFDE2-443D-4BEE-B202-5BFE7706F62B](https://user-images.githubusercontent.com/65950/119282854-210a9000-bc09-11eb-91eb-e60dd400d475.png)

**Comment by albullington** [comment]
@olivierlacan this is where you can find the sharing icon:

<img width="322" alt="Screen Shot 2021-05-23 at 5 58 40 PM" src="https://user-images.githubusercontent.com/35536439/119283086-c9602a80-bbf0-11eb-8ef4-6462fc6342d5.png">

**Comment by olivierlacan** [comment]
Right, but what I'm suggesting is to make sharing available on the species page, not just the new observation notification page.

Because although I'm sure some "sharing intent" occurs when someone is excited to have a new observation, I tend to want to share a species I've discovered in my area, not the fact that I've observed it. More importantly there's no personal information or privacy concern to be had sharing a link to a species page in my mind.

**Comment by albullington** [comment]
Got it. I'm going to edit the title of this feature request to better reflect what I believe you're asking for, which is:

1. The ability for logged-in users to generate a deep link that points to a specific Seek species detail screen
2. The ability to either copy that link or share it directly with social networks

**Comment by olivierlacan** [comment]
Thanks @albullington, I think that does capture my feature request very well. 😃
